document/core/exec/instructions.rst

```diff
@@ -2974,11 +2987,14 @@ The following auxiliary rules define the semantics of entering and exiting :ref:
 
 .. _exec-handler-enter:
 
-Entering an exception handler :math:`H`
-.......................................
+Installing an exception handler :math:`H` containing :math:`\instr^\ast`
```

Comment:
I think I'd keep calling it "entering", since it is a dynamic scope, and you exit it in the end.
```suggestion
Installing an exception handler :math:`H` for :math:`\instr^\ast`
```

Reply:
Actually, it would be more consistent with label blocks to phrase this
```
Entering :math:`\instr^\ast` with exception handler :math:`H`
```

Reply:
I like it! Switching to that.

Reply:
done